src/utils/order.test.ts

```diff
@@ -141,13 +141,70 @@ describe("order utils", () => {
       );
     });
 
-    it("parses CosignedV2DutchOrder", () => {
+    it.only("parses CosignedV2DutchOrder", () => {
       const encodedOrder = cosignedV2DutchOrder.serialize();
       expect(uniswapXOrderParser.parseOrder(encodedOrder, chainId)).toEqual(
         cosignedV2DutchOrder
       );
     });
 
+    it.only("parses CosignedV2DutchOrder 2", () => {
+      // const encodedOrder =
```

Comment:
it fails with this encoded order, which was successfully submitted to the order service